doc/api/tls.markdown

```diff
@@ -615,36 +620,37 @@ Creates a new client connection to the given `port` and `host` (old API) or
 
   - `NPNProtocols`: An array of strings or `Buffer`s containing supported NPN
     protocols. `Buffer`s should have the following format:
-    `0x05hello0x05world`, where first byte is next protocol name's
-    length. (Passing array should usually be much simpler:
-    `['hello', 'world']`.)
+    `0x05hello0x05world`, where the first byte is the next protocol name's
+    length. (Passing an array is usually be much simpler: `['hello', 'world']`.)
 
-  - `ALPNProtocols`: An array of strings or `Buffer`s containing
+  - `ALPNProtocols`: An array of strings or `Buffer`s containing the
     supported ALPN protocols. `Buffer`s should have following format:
     `0x05hello0x05world`, where the first byte is the next protocol
-    name's length. (Passing array should usually be much simpler:
+    name's length. (Passing an array is usually be much simpler:
     `['hello', 'world']`.)
 
-  - `servername`: Servername for SNI (Server Name Indication) TLS extension.
+  - `servername`: Server name for the SNI (Server Name Indication) TLS 
+    extension.
 
   - `checkServerIdentity(servername, cert)`: Provide an override for checking
-    server's hostname against the certificate. Should return an error if verification
-    fails. Return `undefined` if passing.
+    the server's hostname against the certificate. Should return an error if 
```

Comment:
I think it's a little unclear here - do we `return` an error or do we `raise` it? If we return it - what does it describe? 

That can be addressed in a future PR though - this one improves thing anyway.